README.md

```diff
@@ -361,8 +356,8 @@ Why? When a browser opens the client side of your application, this package inte
 -   type: the type of report you want to create inside your `COVERAGE_APP_FOLDER`
 
     -   Default: `coverage`, used to dump the coverage object in a file because when there are several types of test, we want to merge results, and the server reloads between each one.
-    -   Allowed values: `cobertura`, `html`, `json`, `json-summary`, `lcov`, `teamcity`, `text`, `text-lcov`, `text-summary`, `lcovonly`, `coverage`, `remap`
-    -   **Not working values:** `clover`, `cobertura`, `lcov`, `text`, `text-lcov`, PR welcome
+    -   Allowed values: `cobertura`, `coverage`, `html`, `json`, `json-summary`, `lcov`, `lcovonly`, `remap`, `teamcity`, `text`, `text-lcov`, `text-summary`
```

Comment:
The "not working" values are listed in "Allowed values" too. I should remove then `cobertura`, `text-lcov`, `lcov`, `text`.